src/es/components/controllers/favorites/Favorites.js

```diff
@@ -82,7 +82,21 @@ export default class Favorites extends Shadow() {
         detail: {
           fetch: fetch(endpoint, fetchOptions).then(async response => {
             if (response.status >= 200 && response.status <= 299) {
-              window.location = window.location
+              this.dispatchEvent(new CustomEvent('request-basket',
+                {
+                  bubbles: true,
+                  cancelable: true,
+                  composed: true
+                }
+              ))
+              const scrollPosition = document.getElementsByTagName("html")[0].scrollTop + 40 + "px"
+              this.renderNotification("c-favorite", "Ajouté avec succès au panier.", { top: scrollPosition, right: "2em" })
```

Comment:
Is okay, but if it would be a very clean Event Driven Architecture, then you would dispatch an Event here and Notification would be the element which reacts to it and scrolls by itself.